docs/book/src/migration/v3-plugins.md

```diff
@@ -7,5 +7,14 @@ since `28 Apr 2021`.
 Therefore, you can check here how to migrate the projects built from Kubebuilder 3.x with
 the plugin go/v3 to the latest.
 
+## Migration Guides
 
+- [Migration from go/v3 to go/v4](v3vsv4.md)
+- [Migration from go/v3 to go/v4 by updating the files manually](manually_migration_guide_gov3_to_gov4.md)
 
+## Additional Resources
+
+For more information about plugins and their usage, please refer to:
+
+- [Plugins Documentation](../plugins/plugins.md)
+- [Creating Custom Plugins](../plugins/extending.md)
```

Comment:
I think it is enhancement for the docs right?
Could we have a PR for this one with the motivation for it?